docs/core/diagnostics/runtime-loader-binder-events.md

```diff
@@ -0,0 +1,337 @@
+---
+title: "Loader and Binder Runtime Events"
+description: Review loader and binder ETW events, which collect information about the assembly loader and binder.
+ms.date: "09/27/2020"
+ms.topic: reference
+helpviewer_keywords
+  - "Assembly Loader events [.NET Core]"
+  - "Assembly Binder events [.NET Core]"
+  - "ETW, EventPipe, LTTng assembly loader and binder events (CoreCLR)"
+ms.assetid: f2a21e3a-3b6c-4433-97f3-47ff16855ecc
+---
+
+# Loader ETW Events
```

Comment:
H1 headers need to be unique across all of ms docs. This one is used by other articles.